aspnetcore/security/app-secrets.md

```diff
@@ -37,31 +37,34 @@ The Secret Manager tool stores sensitive data for development work outside of yo
 >[!WARNING]
 > The Secret Manager tool does not encrypt the stored secrets and should not be treated as a trusted store. It is for development purposes only. The keys and values are stored in a JSON configuration file in the user profile directory.
 
-### Visual Studio 2017: Installing the Secret Manager tool
+## Installing the Secret Manager tool
+
+# [Visual Studio 2017](#tab/vs2017-sm-tool)
```

Comment:
We've only been approved to use a generic "Visual Studio" tab. It would follow this format:
```
[Visual Studio](#tab/visual-studio)
```
Also, I think the `project.json` content can be deleted.